cloud/pkg/router/utils/path.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"regexp"
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	ParamRegex = "{[-A-Za-z0-9+&@#%?=~_|!:,.;]+}"
```

Comment:
any reference for this regex? any example?

Reply:
add an example before line of the function URLToURLRegex